locked-asset/factory/src/lib.rs

```diff
@@ -1,5 +1,4 @@
 #![no_std]
-#![feature(generic_associated_types)]
 #![feature(exact_size_is_empty)]
```

Comment:
there is a clippy warning from past implementation

Reply:
done